clusterloader/cluster_loader.go

```diff
@@ -156,32 +158,16 @@ func createTemplate(baseName string, ns *v1.Namespace, configPath string, numObj
 		if tuning != nil {
 			if tuning.Templates.RateLimit.Delay != 0 {
 				framework.Logf("Sleeping %d ms between template creation.", tuning.Templates.RateLimit.Delay)
-				time.Sleep(time.Duration(tuning.Templates.RateLimit.Delay) * time.Millisecond)
+				time.Sleep(tuning.Templates.RateLimit.Delay * time.Millisecond)
```

Comment:
If we use `int` we'll still need to convert it to `duration * time.Const` though?